.github/workflows/update-metadata.yml

```
@@ -0,0 +1,29 @@
+name: "Update metadata"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * 0'
+
+permissions:
+  contents: write
+jobs:
+  update-members:
+    name: "Update members.json"
+    runs-on: ubuntu-latest
+    if: github.repository == 'hibernate/.github'
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v6
```

maybe let's have a hash here:
```suggestion
      uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # 6.0.0
```

just to be consistent with what we've been doing in other repos?

---

Sure